src/components/new-request/search.vue

```diff
@@ -1,22 +1,22 @@
 <template>
-  <v-container fluid id="new-request-container" class="copy-normal pa-10">
+  <v-container fluid id="search-container" class="copy-normal pa-10">
     <v-row no-gutters>
       <v-col cols="12" class="pt-0 font-weight-bold h6">
-        I need a name to: {{ request_action_cd }} / {{ location }} / {{ entity_type_cd }}
+        I need a name to: {{ getRequestActionCd }} / {{ getLocation }} / {{ entity_type_cd }}
       </v-col>
     </v-row>
 
-    <v-row class="mt-4" no-gutters>
-      <!-- request_action_cd -->
-      <v-col cols="12" md="4" lg="4">
+    <v-row class="mt-6">
```

Comment:
Needed more top whitespace after I removed vertical padding for all columns (below).

I removed the py for the columns so they would look better when they "wrap", eg:

![image](https://github.com/bcgov/namerequest/assets/10002889/966b91bb-4f65-4c82-a3f5-b8d28deeca0c)